src/Cake.Common/Xml/XmlPokeSettings.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.Text;
+
+namespace Cake.Common.Xml
+{
+    /// <summary>
+    /// Contains settings for <see cref="XmlPokeAliases"/>
+    /// </summary>
+    public sealed class XmlPokeSettings
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="XmlPokeSettings"/> class.
+        /// </summary>
+        public XmlPokeSettings()
+        {
+            Namespaces = new Dictionary<string, string>();
+            Encoding = Encoding.Default;
```

Comment:
As the most common encoding is `UTF8` and Default is unknown (based on system settings) I think `UTF8` should be the "default".